plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/snowflake/TestIcebergSnowflakeCatalogConnectorSmokeTest.java

```diff
@@ -688,8 +689,12 @@ public void testSnowflakeNativeTable()
     @Override
     public void testIcebergTablesFunction()
     {
-        assertThatThrownBy(super::testIcebergTablesFunction)
-                .hasMessageContaining("schemaPath is not supported for Iceberg snowflake catalog");
+        String schemaTablesValues = "VALUES " + getQueryRunner()
```

Comment:
Why do we need to build `VALUES` query? Can we simplify like below? 
```java
        String schemaName = SNOWFLAKE_TEST_SCHEMA.toLowerCase(ENGLISH);
        assertThat(query("SELECT * FROM TABLE(iceberg.system.iceberg_tables(SCHEMA_NAME => '" + schemaName + "'))"))
                .matches("SELECT table_schema, table_name FROM iceberg.information_schema.tables WHERE table_schema = '" + schemaName + "'");
```

Reply:
The `QueryAssert` is way better, let me try that